src/components/Calculator.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import '../App.css';
+
+function Calculator() {
```

Comment:
- [ ] I would advise you to add at least one component to this component, you can do that by extracting the content of this component into another file and then importing it into this file so as to satisfy the requirement.